src/js/contexts/AnnounceContext/doc.js

```diff
@@ -19,9 +19,16 @@ export const doc = AnnounceContext => {
       and these arguments can be passed as 'props' to the return component.
       'mode' can be one of 'polite', 'assertive' or 'off'. 
       'timeout' is measured in milliseconds.
+      'id' used to query a specific Announce Context. 
+      Default is 'grommet-announcer'
       Example:  
       {announce => 
-        <Button onClick={() => announce("Button was clicked", "polite", 1000)
+        <Button onClick={() => announce(
+          "Button was clicked", 
+          "polite", 
+          1000, 
+          "custom-announcer"
```

Comment:
Do we really need this level of flexibility? Can Grommet's AnnounceContext be thought of as a singleton and just use its embedded id automatically?

Reply:
My idea in add this flexibility, instead of let with a fixed ID, is to give user the power to change if is necessary. We never know what user will need.
This doesn't add a lot of code and complexity.
In case you think isn't necessary this flexibility we can let with the default  id only.

Reply:
Understood. We try to keep the API surface of grommet as simple as possible, resisting the urge to build things out because they might be useful someday to someone in the future. Instead, we let practical needs of specific situations drive additions. In this regard, I think we should just fix the issue by adding and using the `grommet-announcer` id. If someone encounters a specific issue that requires further customization in the future, we can add it then.

Reply:
Gotcha. Good to know it. I'm gonna remove it.